iOS-tic-tac-toe/StatusView.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
```

Comment:
Is `Foundation` being used here? I can't recall if `@objc` depends on `Foundation`.